### PR TITLE
[new] new recipe for the NCEP interpolation library

### PR DESCRIPTION
**Pull request by timcera**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timcera** [comment]
@conda-forge-admin, please rerender

**Comment by EricEngle-NOAA** [comment]
I agree to help maintain this package.

**Comment by timcera** [comment]
@conda-forge/staged-recipes, ready for review.

File: recipes/nceplibs-ip/bld.bat [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      -DCMAKE_FIND_APPBUNDLE=NEVER ^
+      -DBUILD_SHARED_LIBS=ON ^
+      -DOPENMP=ON ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+REM Build step
+cmake --build . --config Release
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/nceplibs-ip/bld.bat:21]: Is there are `make check` or `ctest` w can run here?

Comment (EricEngle-NOAA) [reply recipes/nceplibs-ip/bld.bat:21]: You can perform `make test` to run the CMake tests.

Comment (timcera) [reply recipes/nceplibs-ip/bld.bat:21]: There is already a "ctest ..." in build.sh that runs all the tests - I made sure they all passed on my local Linux machine before submitting.

I can add a similar "ctest ..." command for Windows, but can only run the bld.bat script by committing.

I'll put together a commit in a little bit.

Comment (timcera) [reply recipes/nceplibs-ip/bld.bat:21]: The "if" block that is in the "build.sh" to run the "ctest..." line I got from an example somewhere.  Probably just have the "ctest..." command directly in the "bld.bat".

Comment (ocefpaf) [reply recipes/nceplibs-ip/bld.bat:21]: Well, you are skipping Windows. I ended up reviewing this file just b/c it was the first when scrolling.

Comment (timcera) [reply recipes/nceplibs-ip/bld.bat:21]: I thought I might have skipped Windows - but I was confused.  If I skipped Windows, why did a Windows check run?
https://github.com/conda-forge/staged-recipes/pull/22659/checks

Not complaining, just wondering.

Comment (ocefpaf) [reply recipes/nceplibs-ip/bld.bat:21]: > I thought I might have skipped Windows - but I was confused. If I skipped Windows, why did a Windows check run?

Skipping the build doesn't skip the CI here. The CI will always be green though b/c it just starts, check the recipe, and skips it.

**Pull request merged**